crates/kotlin-ffi/src/lib.rs

```diff
@@ -98,6 +103,11 @@ impl From<alloy::providers::utils::Eip1559Estimation> for Eip1559Estimation {
     }
 }
 
+#[derive(uniffi::Record)]
```

Comment:
Had to make minor struct in kotlin-ffi for edge case, but most of the other types are directly exposed to UniFFI in the yttrium crate as we would like